### PR TITLE
Updated docs for systemd

### PR DESCRIPTION
**Pull request by dniewinski**

### CODE REVIEW

**Comment by 130s** [comment]
You're replacing `service` with `systemctrl`. Can you elaborate why this change makes sense? I'm not a super expert in Linux, so tried to understand looking at pages like [this](https://askubuntu.com/questions/903354/difference-between-systemctl-and-service?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa) but I'm just not yet sure if this is an improvement.

**Comment by mikepurvis** [comment]
This looks reasonable to me, but we should probably put this on a new `kinetic-devel` branch, and then call out that `systemctl` is systemd-specific, and refer users to Indigo documentation for the Upstart details.

(Or refactor the whole thing to supply usage documentation alongside the individual plugins...)

File: doc/index.rst [hunk 1]
```diff
@@ -29,15 +29,22 @@ or you can bring it up and down manually:
...
+    $ sudo systemctl status myrobot
 
 If the job is crashing on startup, or you otherwise want to see what is
-being output to the terminal on startup, check the upstart log:
+being output to the terminal on startup, check the systemd log:
 
 .. code-block:: bash
 
-    $ sudo tail /var/log/upstart/myrobot.log -n 30
+    $ sudo journalctl -n 30 -b -u myrobot
```
Comment (paulbovbel) [inline doc/index.rst:47]: It may be worthwhile to note that in Ubuntu, the journal is not persistent across boots by default.

Comment (dniewinski) [reply doc/index.rst:47]: I can add something like this.  Good thing to note

**Comment by dniewinski** [comment]
@130s Ubuntu is changing over from service to systemd in the long-term.  Currently both are supported, but that will change.

**Comment by mikepurvis** [comment]
Well, `usr/sbin/service` is still part of at least Ubuntu Bionic, see: https://packages.ubuntu.com/bionic/all/init-system-helpers/filelist

I'd be kind of surprised if they ever actually removed it, given that it's the common CLI between init systems— whether you're on systemd, upstart, sysvinit.

**Comment by dniewinski** [comment]
I will make a new kinetic-devel branch and store this there.  I'll also make the suggested changed by @paulbovbel

**Pull request closed without merging**